x-pack/plugin/esql/compute/src/test/java/org/elasticsearch/compute/aggregation/ValuesDoubleAggregatorFunctionTests.java

```diff
@@ -37,7 +39,14 @@ protected String expectedDescriptionOfAggregator() {
 
     @Override
     public void assertSimpleOutput(List<Block> input, Block result) {
-        Object[] values = input.stream().flatMapToDouble(b -> allDoubles(b)).boxed().collect(Collectors.toSet()).toArray(Object[]::new);
-        assertThat((List<?>) BlockUtils.toJavaObject(result, 0), containsInAnyOrder(values));
+        TreeSet<?> set = new TreeSet<>((List<?>) BlockUtils.toJavaObject(result, 0));
```

Comment:
Why a TreeSet? Won't it be slower for `containsAll()`?

Reply:
I went with TreeSet so the error output would be sorted.